### PR TITLE
Fixing issue with masonry

### PR DESCRIPTION
**Pull request by tomrndom**
* New function to format the masonry on marketing page
* Adds a new prop "size" to determine if the image occupies a single space or a double spaces on the masonry grid

(same changes as #127 but without merge issues)

**What kind of change does this PR introduce?**

Adds a dummy item without image to render the desired grid

**Does this PR introduce a breaking change?**

Adds a new prop for marketing site masonry.

**What needs to be documented once your changes are merged?**

Every site would need to add the new prop "size" on every image of the sponsor array on marketing-site,json file

ref: https://tipit.avaza.com/project/view#!tab=task-pane&groupby=MyTaskProject&view=vertical&task=2640774&fileview=grid

Signed-off-by: Tomás Castillo <tcastilloboireau@gmail.com>

### CODE REVIEW

File: src/templates/marketing-page.js [hunk 1]
```diff
@@ -37,6 +37,22 @@ export const MarketingPageTemplate = class extends React.Component {
     }
   }
 
+  formatMasonry = (masonry) => {
```
Comment (smarcet) [inline src/templates/marketing-page.js:40]: lets move this to an util file

File: src/templates/marketing-page.js [hunk 2]
```diff
@@ -96,7 +114,7 @@ export const MarketingPageTemplate = class extends React.Component {
               breakpointCols={2}
               className="my-masonry-grid"
               columnClassName="my-masonry-grid_column">
-              {siteSettings.sponsors.map((item, index) => {
+              {formattedMasonry.map((item, index) => {
```
Comment (smarcet) [inline src/templates/marketing-page.js:117]: lets call the utility method here
formatMasonry(siteSettings.sponsors).map(....

File: src/content/marketing-site.json [hunk 3]
```diff
@@ -1,103 +1 @@
-{
```
Comment (smarcet) [inline src/content/marketing-site.json:1]: do we need this file change ?

Comment (tomrndom) [reply src/content/marketing-site.json:1]: I'll format the file, but this change adds the prop `size` for the masonry images

**Pull request merged**